sdk/extensions/chronopost/tests/chronopost/tracking.py

```diff
@@ -0,0 +1,267 @@
+    #         )
+
+    #         self.assertListEqual(lib.to_dict(parsed_response), ParsedErrorResponse)
+
+
+if __name__ == "__main__":
+    unittest.main()
+
+
+TrackingPayload = {
+    "tracking_numbers": ["89108749065090"],
+}
+
+ParsedTrackingResponse = [
+    [
+        {
+            "carrier_id": "chronopost",
+            "carrier_name": "chronopost",
+            "delivered": True,
+            "events": [
+                {
+                    "code": "DC",
+                    "date": "2022-06-19",
+                    "description": "Shipment in preparation to be shipped",
+                    "location": "Web Services",
+                    "time": "13:47",
+                },
+                {
+                    "code": "DB",
+                    "date": "2022-06-20",
+                    "description": "Shipment handed over by shipper",
+                    "location": "AMBOISE",
+                    "time": "16:54",
+                },
+                {
+                    "code": "DY",
+                    "date": "2022-06-20",
+                    "description": "Parcel Registered by the post office",
+                    "location": "AMBOISE",
+                    "time": "16:57",
+                },
+                {
+                    "code": "SC",
+                    "date": "2022-06-21",
+                    "description": "Sorted at departure location",
+                    "location": "TOURS CHRONOPOST",
+                    "time": "19:16",
+                },
+                {
+                    "code": "TS",
+                    "date": "2022-06-21",
+                    "description": "Shipment in transit",
+                    "location": "HUB CHILLY MAZARIN CHRONOPOST",
+                    "time": "23:36",
+                },
+                {
+                    "code": "TS",
+                    "date": "2022-06-22",
+                    "description": "Shipment in transit",
+                    "location": "ARRAS CHRONOPOST",
+                    "time": "02:43",
+                },
+                {
+                    "code": "SD",
+                    "date": "2022-06-22",
+                    "description": "Sorted at delivery location",
+                    "location": "ARRAS CHRONOPOST",
+                    "time": "02:44",
+                },
+                {
+                    "code": "TA",
+                    "date": "2022-06-22",
+                    "description": "Shipment in delivery to the consignee",
+                    "location": "ARRAS CHRONOPOST",
+                    "time": "07:12",
+                },
+                {
+                    "code": "D",
+                    "date": "2022-06-22",
+                    "description": "Delivered",
+                    "location": "ARRAS CHRONOPOST",
+                    "time": "08:30",
+                },
+            ],
+            "tracking_number": "89108749065090",
+        }
+    ],
+    [[]],
```

Comment:
This should just be 
```suggestion
    [],
```

Fixing the error parsing above should correct that